src/frontend/efiling-frontend/src/components/page/home/Home.js

```diff
@@ -41,14 +65,18 @@ const checkCSOAccountStatus = (
 export default function Home({ page: { header, confirmationPopup } }) {
   const [showLoader, setShowLoader] = useState(true);
   const [csoAccountExists, setCsoAccountExists] = useState(false);
+  const [applicantInfo, setApplicantInfo] = useState({});
   const location = useLocation();
   const queryParams = queryString.parse(location.search);
 
-  checkCSOAccountStatus(
-    queryParams.submissionId,
-    setCsoAccountExists,
-    setShowLoader
-  );
+  useEffect(() => {
+    checkCSOAccountStatus(
+      queryParams.submissionId,
+      setCsoAccountExists,
+      setShowLoader,
+      setApplicantInfo
+    );
+  }, [queryParams.submissionId]);
```

Comment:
question: is `queryParams.submissionId` the only thing we need in this [] for useEffect or do we need to specify others as well?

Reply:
This was the only one that came up when running locally.